src/components/PhoneNumberInput/Lazy.js

```diff
@@ -0,0 +1,21 @@
+import { h, Component } from 'preact'
+import style from './style.css'
+
+const Loading = () => <div className={style.loading}>Loading...</div>
+
+class PhoneNumberInputLazy extends Component {
+  constructor(props){
+    super(props)
+    this.state = {component: Loading }
+    // We are loading the mobile validation component dynamically
+    // The code is splitted in two separate chunks.
```

Comment:
Can you explain why we are doing this rather than rather than what/how we are doing it.

https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/